docs/user-manual/en/configuration-index.md

```diff
@@ -481,3 +484,285 @@ Name | Description | Default
 [password](amqp-broker-connections.md#amqp-server-connections) | Broker authentication password (optional) | n/a
 [reconnect-attempts](amqp-broker-connections.md#amqp-server-connections) | How many attempts should be made to reconnect after failure. | -1 (infinite)
 [auto-start](amqp-broker-connections.md#amqp-server-connections) | Broker connection starts automatically with broker | true
+
+
+## Broker Properties Reference
```

Comment:
This list could become stale, could we generate it from the sources?

Reply:
however that turned out to be a huge bit of work
which is going to take a long time to get to  a point where it works and is usable
hence why I updated the schema
as this is where I drive it from